### PR TITLE
Create Cinder app template; integrate ClangFormat

### PR DESCRIPTION
**Pull request by shi-weili**

### CODE REVIEW

File: templates/Bluecadet App/src/_TBOX_PREFIX_App.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		[](SettingsManager *manager) {
+			// Optional: Override json defaults at runtime
+			manager->mFullscreen = false;
+			manager->mWindowSize = ivec2(1280, 720);
+		});
+}
+
+void _TBOX_PREFIX_App::setup() {
+	BaseApp::setup();
+	BaseApp::addTouchSimulatorParams();
```
Comment (benjaminbojko) [inline templates/Bluecadet App/src/_TBOX_PREFIX_App.cpp]: I believe @adielfernandez made this into a JSON parameter. @adielfernandez Did you ever commit this to the views block?

Comment (adielfernandez) [reply templates/Bluecadet App/src/_TBOX_PREFIX_App.cpp]: Yup! The change is in this commit: https://github.com/bluecadet/Cinder-BluecadetViews/commit/2614a01d9589b58209fefba957fbdb3229261494

Comment (benjaminbojko) [reply templates/Bluecadet App/src/_TBOX_PREFIX_App.cpp]: Great! @shi-weili Can you take this line out then and make sure the JSON includes that change. 

@adielfernandez could you do me a favor and grab the SettingsManager + json additions I made to Wetland Data to set the window position and fold it into the views block?

Comment (shi-weili) [reply templates/Bluecadet App/src/_TBOX_PREFIX_App.cpp]: Done.

Comment (adielfernandez) [reply templates/Bluecadet App/src/_TBOX_PREFIX_App.cpp]: @benjaminbojko Sure thing! Will pull the changes from WD and fold them into Views when I get back to NYC.

**Comment by benjaminbojko** [comment]
Great, Weili, can you remove the addTouchParams line then?

**Comment by shi-weili** [comment]
@benjaminbojko the line is removed and code is merged!

**Pull request merged**